tests/Unit/ValueObject/FileValueObjectTest.php

```diff
@@ -42,46 +42,4 @@ public function testFromArray() {
     $this->assertEquals('fr', $file->getLanguageCode());
   }
 
-  /**
-   * Test constructing a file value object from a File entity object.
-   */
-  public function testFromFileEntity() {
-    $language = $this->getMockBuilder(LanguageInterface::class)
-      ->disableOriginalConstructor()
-      ->allowMockingUnknownTypes()
-      ->getMock();
-    $language->expects($this->once())
-      ->method('getId')
-      ->willReturn('fr');
-
-    $file_entity = $this->getMockBuilder(File::class)
-      ->disableOriginalConstructor()
-      ->allowMockingUnknownTypes()
-      ->getMock();
-    $file_entity->expects($this->once())
-      ->method('getFileUri')
-      ->willReturn('http://example.com/test.pdf');
-    $file_entity->expects($this->once())
-      ->method('getMimeType')
-      ->willReturn('pdf');
-    $file_entity->expects($this->once())
-      ->method('getSize')
-      ->willReturn('123');
-    $file_entity->expects($this->once())
-      ->method('getFilename')
-      ->willReturn('Test.pdf');
-    $file_entity->expects($this->once())
-      ->method('language')
-      ->willReturn($language);
-
-    $file = FileValueObject::fromFileEntity($file_entity);
-
-    $this->assertEquals('123', $file->getSize());
-    $this->assertEquals('pdf', $file->getMime());
-    $this->assertEquals('http://example.com/test.pdf', $file->getUrl());
-    $this->assertEquals('Test.pdf', $file->getName());
-    $this->assertEquals('pdf', $file->getExtension());
-    $this->assertEquals('fr', $file->getLanguageCode());
-  }
-
```

Comment:
If we remove this test in favor of a kernel test we should make sure that the kernel test covers all cases that were covered here, such as the language code.